modules/accredited_representative_portal/db/migrate/20241212000200_create_ar_power_of_attorney_forms.rb

```diff
@@ -0,0 +1,14 @@
+class CreateArPowerOfAttorneyForms < ActiveRecord::Migration[7.0]
+  def change
+    create_table :ar_power_of_attorney_forms, id: :uuid do |t|
+      t.references :ar_power_of_attorney_request,
+        type: :uuid,
+        foreign_key: true,
+        index: { name: 'index_ar_poa_forms_on_ar_poa_request_id' }
```

Comment:
`add_reference` indexes by [default](https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference). Do we need this? Was it index name length that prompted the manual specification here?

Reply:
yeah the length went over the limit